src/content/1.7/basics/a-small-tour.md

```diff
@@ -0,0 +1,71 @@
+---
+title: A small tour
+weight: 205
+---
+
+# A small tour of the software
+
+This page attempts to guide newcomers in PrestaShop and show them how to navigate in the software.
+
+Note that a live demo is availalbe on [demo.prestashop.com](https://demo.prestashop.com/).
+
+## Front office MVC
+
+The Front Office is a small application being powered by PHP and Smarty.
+
+The backend relies on controllers you can find in directory `controllers/front`
+and PrestaShop classes that contain the business logic, mainly from `classes` folder.
+
+The views and the Javascript come from the installed theme that you will find in `themes/`. You can have multiple themes available on a shop but only one is enabled and in use.
+
+### Visit a Front controller
+
+When you visit `/` on your shop you are being returned the Homepage.
+
+What happened behind the scenes is that
+
+1. Your HTTP request hit the `index.php` main routing file that relies on the _Dispatcher_
+2. The Dispatcher found the right Front controller ; the `IndexController`
+3. The `IndexController` calls the `index` Smarty view to be rendered, populated with PHP objects
+
+If you send another HTTP request, the same sequence will happen
+- Dispatcher finds the right Front controller
+- Front controller builds the right response using a Smarty template and relying on PrestaShop business logic
+
+## Back office MVC
+
+The Back Office is a large application being powered by PHP and Smarty for the legacy part and by PHP, Symfony and Twig for the migrated part.
```

Comment:
Did we explain somewhere what is the `legacy` and what is `migration`.
In other words, Is the reader aware of the following:
* Prestashop codebase is being migrated to Symfony framework, therefore the keyword "**legacy**" means old, not migrated (and most of the time poor quality) code, while "**migrated**" code is the one that was recently updated following best practices (as much as possible) and advantages of Symfony framework (link to symfony maybe).

Reply:
I moved this page inside Introduction where legacy and migrated are defined.